nibabel/loadsave.py

```diff
@@ -38,6 +38,10 @@ def load(filename, **kwargs):
     '''
     if not op.exists(filename):
         raise FileNotFoundError("No such file: '%s'" % filename)
+
+    if op.getsize(filename) <= 0:
+        raise FileEmptyError("Given file is empty: '%s'" % filename)
```

Comment:
Perhaps we can use `ImageFileError` here, rather than defining a new exception? And I think the message could be shortened to `"Empty file: '%s'" % filename`.